dependencies/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+
+include(FetchContent)
+
+FetchContent_Declare(sqlpp1
+    GIT_REPOSITORY  https://github.com/rbock/sqlpp11
+    GIT_TAG         origin/develop
```

Comment:
The tag specified here, should presumably be origin/master, so it uses a more stable version of sqlpp11 (This is also the way I did with the sqlpp11 connector). Ideally it would probably even point to the latest release, but not sure if it's possible to specify a "latest" tag, so we don't need to manually update the version all the time.
I would say, master is sufficient for now. What would you say @rbock?

Reply:
Yes, usually, master would be the right choice.